frontend/templates/product/index.tsx

```diff
@@ -190,10 +190,12 @@ const ProductTemplate: NextPageWithLayout<ProductTemplateProps> = () => {
                   }
                   case 'DeviceCompatibility':
                      return (
-                        <CompatibilitySection
-                           key={section.id}
-                           compatibility={product.compatibility}
-                        />
+                        !hasCompatibilityNotes && (
+                           <CompatibilitySection
+                              key={section.id}
+                              compatibility={product.compatibility}
+                           />
+                        )
```

Comment:
Instead of hiding this without a replacement, we might consider replacing it with a disclosure or details element that shows some compatibility notes when collapsed, and shows all the notes when expanded.

Reply:
That's what we're doing in the sidebar though right?

Reply:
![image](https://user-images.githubusercontent.com/52104630/232913863-6f00d7f1-805f-4c83-bb18-184c4426bd1a.png)

^ Is this what you mean when you say sidebar?

I was thinking that we could have truncated compatibility notes like in the current sidebar, and then a compatibility notes section that expands/collapses like the current compatibility section:

![image](https://user-images.githubusercontent.com/52104630/232914087-56f6d80c-aac9-4141-9d80-6008b30b2eb9.png)